jetty-quic/quic-common/src/main/java/org/eclipse/jetty/quic/common/QuicConfiguration.java

```diff
@@ -110,4 +104,19 @@ public void setUnidirectionalStreamRecvWindow(int unidirectionalStreamRecvWindow
     {
         this.unidirectionalStreamRecvWindow = unidirectionalStreamRecvWindow;
     }
+
+    public Path getPemWorkDirectory()
+    {
+        return pemWorkDirectory;
+    }
+
+    public void setPemWorkDirectory(Path pemWorkDirectory)
+    {
+        this.pemWorkDirectory = pemWorkDirectory;
+    }
+
+    public Map<String, Object> getImplementationSpecifixContext()
```

Comment:
Rename to `getImplementationConfiguration()`.

Reply:
Done.